packages/wrangler/src/__tests__/secret.test.ts

```diff
@@ -169,10 +171,14 @@ describe("wrangler secret", () => {
         mockAccountId({ accountId: null });
 
         it("should error if a user has no account", async () => {
+          mockGetMemberships({
+            success: false,
+            result: [],
+          });
           await expect(
             runWrangler("secret put the-key --name script-name")
           ).rejects.toThrowErrorMatchingInlineSnapshot(
-            `"No account id found, quitting..."`
+            `"Failed to automatically retrieve account IDs for the logged in user. In a non-interactive environment, it is mandatory to specify an account ID, either by assigning its value to CLOUDFLARE_ACCOUNT_ID, or as \`account_id\` in your \`wrangler.toml\` file."`
```

Comment:
which will make this error be "No account id found, quitting..."

Reply:
The test is occuring in a ["non-TTY" block](https://github.com/cloudflare/wrangler2/pull/852/files/dca02928f3a24413d3272b506bc4ff2d8cc250a2#diff-605262cd36a87ac33a04daefada3d82dd18faf9c9fe53d93854df23bc7648007L137-L138)